### DISCUSSION TITLE
Go To File

### DISCUSSION THREAD

**Discussion by macduy** [community/community#57] [Mobile] [answered]
When Browsing Code in a repo, it would be great to be able to find a file by name, just like the "Go to file" feature on the web interface. Currently I have to navigate to a file by navigating the file structure.

**Comment by rnystrom** [comment, marked as answer]
This is very much on our minds. There is a lot of improvements we are planning to eventually make to file & code browsing. Top of mind:

- "Go to file" search like you mention
- Code search
- Highlighting lines of code when navigating from deep links
- Faster navigation between files and directories (less tapping "back", "back", "back" to get around)

Are there other navigation or file-related features we are missing?

**Reply by macduy** [reply]
Thanks, these all sound great. Code search would be handy too. Looking forward to these features being added to what's an already excellent app.
